### PR TITLE
Fix package generation tests

### PR DESCRIPTION
**Pull request by sladyn98**

### CODE REVIEW

File: src/test/resources/packagerConfig/emptyCascSection.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "bundle": {
     "artifactId" : "jenkins-all-latest",
     "title" : "Jenkins WAR - all latest",
-    "desc" : "New Jenkins Version"
+    "description" : "New Jenkins Version"
   },
   "war" : {
     "jenkinsVersion": "2.107.3"
```
Comment (kwhetstone) [inline src/test/resources/packagerConfig/emptyCascSection.json:8]: Random: want to push jenkinsVersion to the most recent version: 2.248.

Comment (sladyn98) [reply src/test/resources/packagerConfig/emptyCascSection.json:8]: @kwhetston It actually does not matter, since we are just testing the format

**Comment by sladyn98** [comment]
Rebasing and merging

**Comment by sladyn98** [comment]
> Generally, it would be good to include comments on what the tests are doing.

@kwhetstone But doesn't the method name justify the `what` of the tests or should I write it out ?

**Pull request merged**